### PR TITLE
Add aspects to curriculum/javascript/core/flow-control/

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/flow-control/if-statement.md?ref=7be28986a98f7f141ed4f622347208ce51a38c22

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/flow-control/data-type-comparison-in-switch-statements.md: 
 > can not read a block mapping entry; a multiline key may not be an implicit key at line 31, column 1:
    
    ^

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/flow-control/data-type-comparison-in-switch-statements.md: 
 > can not read a block mapping entry; a multiline key may not be an implicit key at line 31, column 1:
    
    ^

File: javascript/core/flow-control/if-statement.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ tags:
 
 aspects:
   - introduction
+  - workout
+  - obscura
```
Comment (ryanwarsaw) [inline javascript/core/flow-control/if-statement.md]: Both of these need to be removed, this is a basic language function.

File: javascript/core/flow-control/data-type-comparison-in-switch-statements.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ links:
 
 
 aspects:
-  - introduction
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/flow-control/data-type-comparison-in-switch-statements.md]: keep `introduction` and remove `deep`

File: javascript/core/flow-control/js-practice-for-loops.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ link: https://glitch.com/edit/#!/remix/x/ff2edbd4-3412-441c-b432-3b348f4e7e01
 answer: /some\sregex/ (optional)
 
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/flow-control/js-practice-for-loops.md]: `introduction`

File: javascript/core/flow-control/js-practice-glitch-for.md [hunk 4]
```diff
@@ -11,8 +11,9 @@ levels:
 tags:
 
   - introduction
+  
 aspects:
-  - introduction
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/flow-control/js-practice-glitch-for.md]: keep `introduction`

File: javascript/core/flow-control/js-practice-glitch-if.md [hunk 5]
```diff
@@ -11,8 +11,9 @@ levels:
 tags:
 
   - introduction
+  
 aspects:
-  - introduction
```
Comment (ryanwarsaw) [inline javascript/core/flow-control/js-practice-glitch-if.md]: `introduction`

File: javascript/core/flow-control/js-practice-switch-case.md [hunk 6]
```diff
@@ -9,6 +9,9 @@ levels:
 
 type: exercise
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/flow-control/js-practice-switch-case.md]: `introduction`

File: javascript/core/flow-control/js-practice-while-loops.md [hunk 7]
```diff
@@ -7,6 +7,9 @@ levels:
 
   - basic
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/flow-control/js-practice-while-loops.md]: `introduction`

File: javascript/core/flow-control/loops-part-1.md [hunk 8]
```diff
@@ -20,8 +20,11 @@ tags:
 
   - introduction
 
+
 aspects:
   - introduction
+  - workout
+  - obscura
```
Comment (ryanwarsaw) [inline javascript/core/flow-control/loops-part-1.md]: remove `obscura`

File: javascript/core/flow-control/loops-part-2.md [hunk 9]
```diff
@@ -19,8 +19,11 @@ tags:
 
   - introduction
 
+
 aspects:
   - introduction
+  - workout
+  - obscura
```
Comment (ryanwarsaw) [inline javascript/core/flow-control/loops-part-2.md]: remove `obscura`

**Pull request merged**